packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -36,8 +36,8 @@ function useAddAppMutation(_type: App["type"] | null, options?: UseAddAppMutatio
       isOmniInstall = variables.isOmniInstall;
       type = variables.type;
     }
-    if (type === "sendgrid_other_calendar") {
-      type = "sendgrid";
+    if (type?.endsWith("_other_calendar")) {
+      type = type.split("_other_calendar")[0];
```

Comment:
@hariombalhara making this as a general rule instead for specific apps as I renamed all other `_other_calendar` apps within this PR.